virt/backup_restore/virt-backup-restore-overview.adoc

```diff
@@ -6,8 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you back up and restore {VirtProductName} virtual machines (VMs) by using the OpenShift API for Data Protection (OADP).
-// Non-admin user [https://issues.redhat.com/browse/OADP-203] is targeted for OADP 1.1.
+You back up and restore {VirtProductName} virtual machines (VMs) by using the OpenShift API for Data Protection (OADP).
```

Comment:
I still think you need to distinguish the use cases. 

As a cluster admin, you install the Operator. 
As a "X" (you need to find out what this role actually is), you back up and restore your VMs in your namespace.